src/SpellsReference/Controllers/SpellController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using SpellsReference.Data;
+using SpellsReference.Data.Repositories;
+using SpellsReference.Models;
+using System.Collections.Generic;
+using System.Web.Mvc;
+
+namespace SpellsReference.Controllers
+{
+    [Authorize]
+    public class SpellController : Controller
+    {
+        private IContext context;
+
+        public SpellController(IContext context)
+        {
+            this.context = context;
+        }
```

Comment:
We should inject the repository dependency here, instead of creating new instances of the repository for the controller actions.

Reply:
Also, the repositories use `_context` instead of `context`. Perhaps we should stick to one style.